test/exifviewer.jl

```diff
@@ -0,0 +1,89 @@
+@testset "exifviewer.jl" begin
+
+    @testset "Basic IO" begin
+        io = open(filepath, "r")
+        @test read_tags(io; read_all=true)["EXIF_TAG_PIXEL_X_DIMENSION"] == "3000"
+        close(io)
```

Comment:
Also nitpicking: use `open(filepath, "r") do io ... end` syntax here and below -- any explicit `close` gives a smell of bad practice.

Let me try to re-explain this a bit:

```julia
io = open(...)
do_something(io)
close(io)
```

if `do_something` fails and errors, then `close(io)` will not be called at all, which means you'll never release the resource unless you restart Julia. This is why we need

```julia
io = open(...)
try
    do_something(io)
finally
    close(io)
end
```

and the above is simplified to

```julia
open(...) do io
    do_something(io)
end
```

which is backed by:

```julia
function open(f::Function, args...)
    io = open(args...)
    try
        f(io)
    finally
        close(io)
    end
end
```